docs/HowToDebugCrashWithAddr2lineTool.md

```diff
@@ -0,0 +1,116 @@
+# How to use Addr2Line tool for debugging
```

Comment:
Please change the link in `HowToDebug.md.`
It points the `HowToUseAddr2Line.md` now.

Reply:
Done